x/subscription/keeper/subscription.go

```diff
@@ -217,13 +156,6 @@ func (k Keeper) CreateSubscription(
 	sub.DurationTotal = duration
```

Comment:
doesnt this needs to be += duration?

Reply:
`DurationTotal` is the total duration **last requested**. 
It is/will be useful for auto-renewal, so whoever does auto-renewal (be it the chain or a 3rd part) will be able to repeat the last subscription duration. For instance, a user that buys a 1-year subscription with discount, usually expects it to auto-renew by 1-year each time.